### PR TITLE
Fix lightpush test

### PR DESCRIPTION
**Pull request by oskarth**
TEst

### CODE REVIEW

File: tests/v2/test_wakunode.nim [hunk 1]
```diff
@@ -568,67 +568,66 @@ procSuite "WakuNode":
...
-  #     (await completionFutRelay.withTimeout(5.seconds)) == true
-  #     (await completionFutLightPush.withTimeout(5.seconds)) == true
-  #   await node1.stop()
-  #   await node2.stop()
-  #   await node3.stop()
+  asyncTest "Lightpush message return success":
+    let
+      nodeKey1 = crypto.PrivateKey.random(Secp256k1, rng[])[]
+      node1 = WakuNode.init(nodeKey1, ValidIpAddress.init("0.0.0.0"),
+        Port(60000))
```
Comment (oskarth) [inline tests/v2/test_wakunode.nim:575]:
```suggestion
        Port(60010))
```

File: tests/v2/test_wakunode.nim [hunk 2]
```diff
@@ -568,67 +568,66 @@ procSuite "WakuNode":
...
+      nodeKey2 = crypto.PrivateKey.random(Secp256k1, rng[])[]
+      node2 = WakuNode.init(nodeKey2, ValidIpAddress.init("0.0.0.0"),
+        Port(60002))
```
Comment (oskarth) [inline tests/v2/test_wakunode.nim:578]:
```suggestion
        Port(60012))
```

File: tests/v2/test_wakunode.nim [hunk 3]
```diff
@@ -568,67 +568,66 @@ procSuite "WakuNode":
...
+      nodeKey3 = crypto.PrivateKey.random(Secp256k1, rng[])[]
+      node3 = WakuNode.init(nodeKey3, ValidIpAddress.init("0.0.0.0"),
+        Port(60003))
```
Comment (oskarth) [inline tests/v2/test_wakunode.nim:581]:
```suggestion
        Port(60013))
```

**Comment by oskarth** [comment]
See https://github.com/status-im/nim-waku/issues/515 closing this PR.

File: tests/v2/test_wakunode_light.nim [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+      node3 = WakuNode.init(nodeKey3, ValidIpAddress.init("0.0.0.0"),
+        Port(60013))
+      pubSubTopic = "test"
+      contentTopic = ContentTopic("/waku/2/default-content/proto")
+      payload = "hello world".toBytes()
+      message = WakuMessage(payload: payload, contentTopic: contentTopic)
+
+    # Light node, only lightpush
+    await node1.start()
+    node1.mountLightPush()
```
Comment (jm-clius) [inline tests/v2/test_wakunode_light.nim:45]: Adding
```
node2.mountRelay(relayMessages=false) # Mount WakuRelay, but do not start or subscribe to any topics
```
between lines 44 and 45 should fix the CI

**Pull request closed without merging**